dev-packages/e2e-tests/test-applications/angular-17/src/app/app.routes.ts

```diff
@@ -11,6 +11,18 @@ export const routes: Routes = [
     path: 'home',
     component: HomeComponent,
   },
+  {
+    path: 'redirect1',
+    redirectTo: '/redirect2',
+  },
+  {
+    path: 'redirect2',
+    redirectTo: '/redirect3',
+  },
+  {
+    path: 'redirect3',
+    redirectTo: '/users/456',
+  },
```

Comment:
Although these are 3 redirects, we'll only get one navigation root span and one routing span for them. I think this is fine (3 routing spans would have been nice to have but whatever)